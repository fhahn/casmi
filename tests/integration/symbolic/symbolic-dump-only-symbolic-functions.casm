function a : -> Int initially { 2 }
function(symbolic) b : -> Int

init main

rule main = {
    a := 1
    assert symbolic( b )
    program(self) := undef
}
