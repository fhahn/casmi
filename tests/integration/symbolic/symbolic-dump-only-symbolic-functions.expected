forklog:
tff(symbolNext, type, sym1: $int).
fof(id1,hypothesis,stb(1,sym1)).%CREATE: b
fof(id2,hypothesis,stb(1,sym1)).%SYMBOLIC: b
fof(final0,hypothesis,stb(0,sym1)).%FINAL: b

